File: sample-videochat-webrtc/sample-videochat-webrtc/Components/Users/Views/TitleView/TitleView.m
//
//  TitleView.m
//  sample-videochat-webrtc
//
//  Created by Injoit on 1/31/20.
//  Copyright © 2020 Quickblox. All rights reserved.
//

#import "TitleView.h"

@implementation TitleView
//MARK: - Life Cycle
- (instancetype)initWithFrame:(CGRect)frame {
    self = [super initWithFrame:frame];
    if (self) {
        self.textColor = UIColor.whiteColor;
        self.font = [UIFont systemFontOfSize:17.0f weight:UIFontWeightBold];
        self.textAlignment = NSTextAlignmentCenter;
    }
    return self;
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/Users/Views/TitleView/TitleView.swift
//
//  TitleView.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 10/10/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

import UIKit

class TitleView: UILabel {
    var title = "" {
        didSet {
            text = title
        }
    }

    override init(frame: CGRect) {
        super.init(frame: .zero)

        font = .systemFont(ofSize: 17.0, weight: .bold)
        textColor = .white
        textAlignment = .center
        lineBreakMode = .byTruncatingTail
    }
    
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
}
